scripts/lint/testLint.ts

```diff
@@ -5,7 +5,6 @@
 
 import glob from 'glob'
 import Mocha from 'mocha'
-
 ;(async () => {
```

Comment:
?

Reply:
will revert this, not sure where is this change coming from

Reply:
@justinmk3 this change seems to be made by linter automatically when I make a commit